src/test/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceServiceTest.java

```diff
@@ -134,98 +146,109 @@ public void getObjects() throws Exception {
     }
 
     @Test
-    public void findByQueryParametersStringParameters() throws Exception
-    {
+    public void findByQueryParameters_ObjectUrnLike() throws Exception {
         populateQueryData();
 
-        Map<String, Object> queryParams = new HashMap<>();
-
-        queryParams.put(ObjectPersistenceService.OBJECT_URN_LIKE, "objectUrnQueryParams");
+        Map<QueryParameterType, Object> queryParams = new HashMap<>();
+        int expectedSize = 0;
+        int actualSize = 0;
+
+        expectedSize = 12;
```

Comment:
I have found that while this way is more verbose it provides much more useful information later when a test fails.